mypy/fixup.py

```diff
@@ -290,12 +290,14 @@ def lookup_qualified_stnode(modules: Dict[str, MypyFile], name: str,
         names = node.names
 
 
-def stale_info() -> TypeInfo:
+def stale_info(modules: Dict[str, MypyFile]) -> TypeInfo:
     suggestion = "<stale cache: consider running mypy without --quick>"
     dummy_def = ClassDef(suggestion, Block([]))
     dummy_def.fullname = suggestion
 
     info = TypeInfo(SymbolTable(), dummy_def, "<stale>")
-    info.mro = [info]
-    info.bases = []
+    obj_type = lookup_qualified(modules, 'builtins.object', False)
+    assert isinstance(obj_type, TypeInfo)
+    info.bases = [Instance(obj_type, [])]
```

Comment:
Is this really necessary?

Reply:
Yeah, `join` looks explicitly at `bases`.